### ISSUE TITLE
Now working on Mac

### ISSUE THREAD

**Issue by derberg** [github/VisualStudio#1880]
## Version

- GitHub Extension for Visual Studio version: lates from the official site
- Visual Studio version: 1.26.1 on Mac

## What happened

**Steps to Reproduce**

I try to install extension on my VS on Mac and get this error https://github.com/github/VisualStudio/issues/876 

Is it still valid that extension work on Windows only? if yes, they why it is not clearly stated on the extension landing page? I would save a lot of time if that would be clearly communicated

**Comment by jcansdale** [comment]
@derberg,

> Is it still valid that extension work on Windows only? if yes, they why it is not clearly stated on the extension landing page? I would save a lot of time if that would be clearly communicated

I'm afraid the extension currently only works on Visual Studio for Windows not Visual Studio Code. When we started developing the extension there was no such things and Visual Studio Code or Visual Studio for Mac. It definitely makes sense to update the website to clarify which Visual Studio version is works with.

I've updated the extension to explicitly state:
"Requires Windows and Visual Studio 2015 or newer"

I couldn't find an obvious way to make Visual Studio Code reject a Visual Studio .vsix file. I'll ask around about how to do this.
